### DISCUSSION TITLE
rbfly -  a library for RabbitMQ Streams using Python asyncio

### DISCUSSION THREAD

**Discussion by wrobell** [rabbitmq/rabbitmq-server#3999] [Other]
I have just released first version of RbFly. It is a library for RabbitMQ Streams using Python asyncio

 https://wrobell.dcmod.org/rbfly/index.html

The library is designed and implemented with the following features in mind

1. Simple and flexible API.
2. Use of AMQP 1.0 message format to enable interoperability between RabbitMQ Streams clients.
3. Performance by implementing critical sections of code using Cython, avoiding copying of data, and testing. The testing involves use of asyncio drop-in replacement uvloop, which improves performance of Python asyncio based applications.
4. Auto reconnection to RabbitMQ broker with lazily created connection objects.

**Comment by acogoluegnes** [comment]
@wrobell Thanks for letting the community know and good job! :-)

I would suggest to post also on the [RabbitMQ Users mailing list](https://groups.google.com/g/rabbitmq-users), you should reach a large number of people.
